libraries/Microsoft.Bot.Builder/BotFrameworkAdapter.cs

```diff
@@ -828,13 +828,13 @@ public virtual async Task CreateConversationAsync(string channelId, string servi
             if (reference.Conversation != null)
             {
                 var typeOfDynamic = reference.Conversation.GetType();
-                var tenantProperty = typeOfDynamic.GetProperty("tenantId");
+                var tenantProperty = typeOfDynamic.GetProperty("TenantId");
```

Comment:
>TenantId [](start = 64, length = 8)

This is totally not right.
Why are you going through reflection?  
This is a typed object:  You should be using 
           reference.Conversation.TenantId

Reply:
@tomlm I thought that was odd, too, but assumed that since it's been in the SDK this way for awhile, it may have been intentional. I'll do some testing with it today and fix it up.

Reply:
@tomlm Fixed.